src/components/GeneratorHeader/GeneratorHeader.tsx

```diff
@@ -1,13 +1,18 @@
 import Link from 'next/link';
 import React from 'react';
 import { linkToGenerator, linkToSearch } from '../../helpers/links';
+import styles from './GeneratorHeader.module.scss';
 
 function GeneratorHeader() {
     return (
         <header>
-            <nav>
-                <Link href={linkToGenerator()}>Generator</Link>
-                <Link href={linkToSearch()}>Search</Link>
+            <nav className={styles.root}>
+                <Link className={[styles.activeTab, styles.tab].join(' ')} href={linkToGenerator()}>
```

Comment:
You are assuming here that generator is always the active tab. You can check it using router. https://nextjs.org/docs/api-reference/next/router You will have to use `useRouter` hook and then check if `router.pathname` is equal to the link url.